### PR TITLE
Bitcoind/v24

### PR DESCRIPTION
**Pull request by islandbitcoin**

### CODE REVIEW

File: manifest.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 id: c-lightning
-version: 0.12.0.1
+version: 0.12.0.2
```
Comment (chrisguida) [inline manifest.yaml:2]: this is unncessary, please revert

File: scripts/services/migrations.ts [hunk 2]
```diff
@@ -81,5 +81,5 @@ export const migration: T.ExpectedExports.migration = compat.migrations
         ),
       },
     },
-    "0.12.0.1",
+    "0.12.0.2",
```
Comment (chrisguida) [inline scripts/services/migrations.ts:84]: This is also unnecessary

**Review by chrisguida — CHANGES_REQUESTED** [review]
see change requests

File: scripts/services/migrations.ts (hunk 2 above)
Comment (islandbitcoin) [reply scripts/services/migrations.ts]: fixed

File: manifest.yaml (hunk 1 above)
Comment (islandbitcoin) [reply manifest.yaml]: fixed

**Review by chrisguida — APPROVED** [review]
Looks good

**Pull request merged**